src/coreclr/nativeaot/Runtime/AsmOffsetsVerify.cpp

```diff
@@ -1,15 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
+
 #include "common.h"
 #include "gcenv.h"
 #include "gcheaputilities.h"
-#include "rhassert.h"
+#include "aotassert.h"
```

Comment:
`aotassert.h` -> `debugmacros.h` so that it is aligned with a file that serves similar purpose in regular CoreCLR